Src/Fare.IntegrationTests/XegerTests.cs

```diff
@@ -87,7 +127,7 @@ public void GeneratedTextIsCorrectWithRexEngine(string pattern)
             "x[0-9A-Z]",
             "[^A-M]in",
             ".gr",
-            @"\(.*l",
+            //@"\(.*l", - on "classic" went sometimes overflow (even before my changes)
```

Comment:
Can you put it back? (Uncomment) We can always tackle the [erratic](http://xunitpatterns.com/Erratic%20Test.html) tests in a separate PR.